app/scripts/modules/core/src/pipeline/config/stages/pipeline/pipelineStage.js

```diff
@@ -93,6 +93,11 @@ module.exports = angular
               $scope.stage.pipelineParameters = {};
             }
             $scope.pipelineParameters = config.parameterConfig;
+            $scope.pipelineParameters.forEach(parameterConfig => {
+              if (parameterConfig.default && parameterConfig.options && !parameterConfig.options.some(option => option.value === parameterConfig.default)) {
+                parameterConfig.options.unshift({ value: parameterConfig.default });
```

Comment:
doesn't need a `label`?

Reply:
Nope. `ui-select` deals with objects and we decide however we want to render the object.

In this particular case, we're just doing
```
<div><strong>{{option.value}}</strong></div>
```
so no need to duplicate `value` and `label`.